### DISCUSSION TITLE
Docker container updated but YOURLS version not updated?

### DISCUSSION THREAD

**Discussion by EDIflyer** [YOURLS/YOURLS#3544] [Q&A] [answered]
I saw the new 1.9.2 release out to day and updated my YOURLS Docker image. I restarted the container after pulling the new image. It all seems to still be running OK however it is still showing v.1.9.1 and asking me to upgrade.

Inspecting the Docker container looks like it has updated to 1.9.2 OK:
![image](https://user-images.githubusercontent.com/13610277/223573804-871269cc-a4ed-410f-b321-d7ad05007607.png)
![image](https://user-images.githubusercontent.com/13610277/223574341-606d90e6-6994-469c-a169-5d6d41ee932f.png)

However still getting the upgrade message:
![image](https://user-images.githubusercontent.com/13610277/223573873-271f511d-8210-47fb-aee7-6edebeac48ee.png)
(and 1.9.1 at the bottom of the page - I also note it now shows a PHP warning message at the top although it seemed to disappear when refreshing the page)
![image](https://user-images.githubusercontent.com/13610277/223573975-afad92b2-0931-45d5-91ba-86c653984694.png)

Docker Hub is showing the latest version as being updated earlier today - https://hub.docker.com/layers/library/yourls/latest/images/sha256-37273c858a744adfdeb72785a8ae80a3b0a7cfa4426960ebb7967f7b998ceaf4?context=explore

**Comment by mvthde** [comment]
I experience the same behavior.

**Reply by EDIflyer** [reply]
Thanks for confirming. Interestingly when I did a fresh install on another server it showed 1.9.2 OK, so not sure if there's some issue where it's not upgrading properly?

**Comment by LeoColomb** [comment, marked as answer]
Thanks for your report, @EDIflyer @mvthde.
Do you have a volume mounted?

**Reply by EDIflyer** [reply]
Yes I do @LeoColomb I've got `/var/lib/mysql` for the DB and `/var/www/html/user/plugins` and `/var/www/html` on the YOURLS container itself  (although in retrospect I guess the plugins one is superfluous given its just in a subdirectory!)

**Reply by LeoColomb** [reply]
Then this is the explanation: any file in your volume `/var/www/html` is going to take precedence to new application file, to avoid any unexpected override. Thus, the previous 1.9.1 files are still used.
You can upgrade by either dropping your volume (probably not if it's here), or perform the upgrade yourself.

**Reply by EDIflyer** [reply]
Ah, very interesting, thanks @LeoColomb. I think I'd mapped the rest just to view config, add plugins and to add an index.html file to root, but if I just map that one individual index.html file and the plugin folder then that should let it use the 1. 9.2 files?

**Reply by mvthde** [reply]
Yes, I've tested it and that works @EDIflyer . Thank's @LeoColomb !

**Reply by EDIflyer** [reply]
Thanks from me too, @LeoColomb - I changed my docker-compose to map just the following your the yourls containers and it has updated perfectly. Many thanks for your help!!
```
      - /home/ediflyer/containers/yourls/plugins:/var/www/html/user/plugins
      - /home/ediflyer/containers/yourls/index.html:/var/www/html/index.html
      - /home/ediflyer/containers/yourls/bg.jpg:/var/www/html/bg.jpg
```
